### DISCUSSION TITLE
Easter Eggs

### DISCUSSION THREAD

**Discussion by ShankarBUS** [ShankarBUS/ModernFlyouts#295] [Ideas]
How about adding some hidden features within our app.

Like the rainbow reveal or something.

I had some suprise plans for v0.10 which included Easter eggs (such as themes for a festival or something).

What do guys have in mind?
Should we do these or focus on making the app stable and opt out of preview (i.e. to hit v1.0)?

**Comment by prakharb5** [comment]
Well, TBH, discussing these features here will make these features not secret, sadly 😂.
Well, one or two features like these won't hurt. But main aim should be on reaching v1.

About the rainbow effect, if it is ready, you can ship it with the latest version (0.8.5 as of now). Would be great to see!

**Reply by ShankarBUS** [reply]
Darn it! I completely forgot about it!

**Reply by ShankarBUS** [reply]
But only active users on GitHub will see this.

We have 54k+ users. Most of them won't look at this GitHub repository though.
So, we can talk.

**Reply by prakharb5** [reply]
I was just telling 😁. No probs!

**Comment by Renzo904** [comment]
I think it would be a great idea hide only a few easter eggs now(or only one, i don't know how many ideas do you have). 
And after v1.0 start adding the others easter eggs.

**Reply by ShankarBUS** [reply]
I just had only one in mind.
Maybe future contributors can expand the list 🤷‍♂️.

**Reply by Renzo904** [reply]
Yeah maybe, IMO i would add that easter egg, and then wait for the v1 to see how it goes

**Reply by prakharb5** [reply]
@ShankarBUS 
Give rainbow reveal now!

**Comment by Samuel12321** [comment]
I personally think we need to focus on stability for 0.9.x and 0.10 releases. 0.9 seemed to introduce a few bugs.
